core/src/main/java/com/linecorp/armeria/internal/ArmeriaHttpUtil.java

```diff
@@ -334,6 +334,25 @@ public static String decodePath(String path) {
         return new String(buf, 0, dstLen, StandardCharsets.UTF_8);
     }
 
+    /**
+     * Returns {@code true} if the specified {@code path} is an absolute {@code URI}.
+     */
+    public static boolean isAbsolutePath(@Nullable String path) {
```

Comment:
`isAbsoluteUri()`?

Reply:
I was considering that name too. :-)
Does it need to change the parameter name or it is fine?

Reply:
`pathOrUri` or `maybeUri`?